etcdctl/command/member_commands.go

```diff
@@ -132,8 +130,10 @@ func actionMemberRemove(c *cli.Context) {
 	removalID := args[0]
 
 	mAPI := mustNewMembersAPI(c)
+
+	totalCtx, _ := contextWithTotalTimeout(c)
 	// Get the list of members.
-	listctx, listCancel := context.WithTimeout(context.Background(), client.DefaultRequestTimeout)
+	listctx, listCancel := context.WithTimeout(totalCtx, client.DefaultRequestTimeout)
```

Comment:
there is no need to set client.DefaultRequestTimeout here because user doesn't even know it.